ui/src/pages/Discord/Servers/Bunzone.tsx

```diff
@@ -0,0 +1,9 @@
+import { ServerTemplate } from "./ServerTemplate/ServerTemplate";
+
+export function Bunzone() {
+  return(
+    <>
+      <ServerTemplate title="The Bunzone" description="" img="bunnyL.jpg" joinLink="discord/bruh" />
+    </>
```

Comment:
Get rid of <> and </> and write line 6 as:
```suggestion
    <ServerTemplate
      title="The Bunzone"
      description=""
      img="bunnyL.jpg"
      joinLink="discord/bruh"
    />
```